pkg/result/processors/severity.go

```diff
@@ -19,6 +19,10 @@ type severityRule struct {
 	severity string
 }
 
+// Severity modifies report severity.
+// It uses the same [baseRule] structure as [ExcludeRules] processor.
```

Comment:
Can be
```suggestion
// It uses the same baseRule structure as [ExcludeRules] processor.
```

because pkgsite doesn't render links for unexported symbols:

<img width="859" alt="image" src="https://github.com/user-attachments/assets/a3122d12-7717-4ccf-906e-3fba3281d239" />